README.md

```diff
@@ -1,26 +1,4 @@
-<!--
-
-    TODO:
-
-    - Add the project to the CircleCI:
-      https://circleci.com/setup-project/gh/giantswarm/REPOSITORY_NAME
-
-    - Import RELEASE_TOKEN variable from template repository for the builds:
-      https://circleci.com/gh/giantswarm/REPOSITORY_NAME/edit#env-vars
-
-    - Change the badge (with style=shield):
-      https://circleci.com/gh/giantswarm/REPOSITORY_NAME/edit#badges
-      If this is a private repository token with scope `status` will be needed.
-
-    - Change the top level header from `# template` to `# REPOSITORY_NAME` and
-      add appropriate description.
-
-    - If the repository is public consider adding godoc badge. This should be
-      the first badge separated with a single space.
-      [![GoDoc](https://godoc.org/github.com/giantswarm/REPOSITORY_NAME?status.svg)](http://godoc.org/github.com/giantswarm/REPOSITORY_NAME)
-
--->
-[![CircleCI](https://circleci.com/gh/giantswarm/template.svg?style=shield&circle-token=cbabd7d13186f190fca813db4f0c732b026f5f6c)](https://circleci.com/gh/giantswarm/template)
+[![CircleCI](https://circleci.com/gh/giantswarm/k8s-api-healthz.svg?style=shield&circle-token=cbabd7d13186f190fca813db4f0c732b026f5f6c)](https://circleci.com/gh/giantswarm/k8s-api-healthz)
 
 # k8s api healthz
 This is simple service thats suppose to be running on the master node in order to properly and securely check health of the API.
```

Comment:
It would be nice to have an explanation here how the health check works.

- What endpint is accessed in this service?
- What does it check in the kubernetes API?

Reply:
you probably need to read whole Readme, not just the diff which is not about the whole description
https://github.com/giantswarm/k8s-api-healthz/blob/master/README.md